src/components/AddItem.js

```diff
@@ -19,7 +19,7 @@ const AddItem = ({ token, results }) => {
       writeToFirestore(token, {
         name,
         frequency,
-        lastPurchaseDate: null,
+        purchaseDates: [],
```

Comment:
It took me a bit to find a list with this active in firebase but it was awesome once I did. Looks really good. Not sure how to test the 24 hr part tho.

Reply:
Use "gunk wispy jay"!